### PR TITLE
Update basic SQL operations

### PR DESCRIPTION
**Pull request by ireneontheway**
split to DDL DML DQL DCL sections.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: [pingcap/docs-cn#3215](https://github.com/pingcap/docs-cn/pull/3215)
- Other reference link(s):

### CODE REVIEW

**Comment by ireneontheway** [comment]
PTAL @yikeke

File: basic-sql-operations.md [hunk 1]
```diff
@@ -207,71 +178,68 @@ To insert data into a table, use the `INSERT` statement:
...
 ```
 
-```
-+--------+------+------------+
-| number | name | birthday   |
-+--------+------+------------+
-|      1 | tom  | 2017-10-10 |
-+--------+------+------------+
-```
+Note: UPDATE and DELETE without WHERE filter operate on the entire table.
```
Comment (ireneontheway) [inline basic-sql-operations.md:205]: The expression of "note" in pingcap/doc seems not to follow the guideline.

File: basic-sql-operations.md [hunk 2]
```diff
@@ -178,25 +150,27 @@ To show all the indexes in a table, use the `SHOW INDEX` statement:
...
-ALTER TABLE person DROP INDEX person_num;
+ALTER TABLE person DROP INDEX person_unique_id;
 ```
 
-## Insert, select, update, and delete data
+> **Note:**
+> 
+> DDL operations are not transactions. You don't need corresponding COMMIT statements when executing DDL.
+>
+> Common DML features are adding, modifying, and deleting tables. The corresponding commands are CREATE,  ALTER,and DELETE.
```
Comment (ran-huang) [inline basic-sql-operations.md:171]:
```suggestion
> Common DML features are adding, modifying, and deleting table records. The corresponding commands are `INSERT`, `UPDATE`, and `DELETE`.
```
btw, I think this sentence should not be included in the **Note**. What do you think?

File: basic-sql-operations.md [hunk 3]
```diff
@@ -178,25 +150,27 @@ To show all the indexes in a table, use the `SHOW INDEX` statement:
...
 
-### Insert data
+## Insert, select, update, and delete data
```
Comment (ran-huang) [inline basic-sql-operations.md:173]:
```suggestion
## Insert, update, and delete data
```

File: basic-sql-operations.md [hunk 4]
```diff
@@ -206,71 +180,70 @@ To insert data into a table, use the `INSERT` statement:
 INSERT INTO person VALUES("1","tom","20170912");
 ```
 
-### Select data
-
-To view the data in a table, use the `SELECT` statement:
+To insert data into a table containing data, use the `INSERT` statement:
```
Comment (ran-huang) [inline basic-sql-operations.md:183]:
```suggestion
To insert data into a table containing data of some fields, use the `INSERT` statement:
```

File: basic-sql-operations.md [hunk 5]
```diff
@@ -206,71 +180,70 @@ To insert data into a table, use the `INSERT` statement:
...
-+--------+------+------------+
-| number | name | birthday   |
-+--------+------+------------+
-|      1 | tom  | 2017-09-12 |
-+--------+------+------------+
-```
-
-### Update data
-
 To update the data in a table, use the `UPDATE` statement:
```
Comment (ran-huang) [inline basic-sql-operations.md:191]:
```suggestion
To update some fields of a record in a table, use the `UPDATE` statement:
```

File: basic-sql-operations.md [hunk 6]
```diff
@@ -206,71 +180,70 @@ To insert data into a table, use the `INSERT` statement:
...
-```
-+--------+------+------------+
-| number | name | birthday   |
-+--------+------+------------+
-|      1 | tom  | 2017-10-10 |
-+--------+------+------------+
-```
+> **Note:**
+> 
+> UPDATE and DELETE without WHERE filter operate on the entire table.
```
Comment (ran-huang) [inline basic-sql-operations.md:209]:
```suggestion
> The `UPDATE` and `DELETE` statements without the `WHERE` clause as a filter operate on the entire table.
```

File: basic-sql-operations.md [hunk 7]
```diff
@@ -206,71 +180,70 @@ To insert data into a table, use the `INSERT` statement:
...
+>
+> DQL is used to retrieve the desired data rows from a table or multiple tables, usually the core content.
```
Comment (ran-huang) [inline basic-sql-operations.md:211]: Similarly, this sentence doesn't look like a part of this **note**. Rather, it reads more like an introduction of the next section `query data`

File: basic-sql-operations.md [hunk 8]
```diff
@@ -206,71 +180,70 @@ To insert data into a table, use the `INSERT` statement:
...
++------+
+| name |
++------+
+| tom  |
++------+
 ```
 
-```
-Empty set (0.00 sec)
+Use the WHERE clause to filter all records for eligibility and then return:
```
Comment (ran-huang) [inline basic-sql-operations.md:236]:
```suggestion
Use the `WHERE` clause to filter all records that match the conditions and then return the result:
```

**Comment by ireneontheway** [comment]
@ran-huang PTAL

File: basic-sql-operations.md [hunk 9]
```diff
@@ -206,71 +180,70 @@ To insert data into a table, use the `INSERT` statement:
...
+Use the `WHERE` clause to filter all records that match the conditions and then return the result:
+
+{{< copyable "sql" >}}
+
+```sql
+SELECT * FROM person where id<5;
 ```
 
-## Create, authorize, and delete a user
+Common DCL features are creating or deleting users, and managing user privileges.
```
Comment (ran-huang) [inline basic-sql-operations.md:244]:
```suggestion
DCL are usually used to create or delete users, and manage user privileges.
```

**Comment by ireneontheway** [comment]
Thanks a lot. I've updated it according to your advice. @ran-huang

**Comment by ran-huang** [comment]
@pepezzzz PTAL. Thanks~

File: basic-sql-operations.md [hunk 10]
```diff
@@ -1,174 +1,156 @@
 ---
-title: Explore SQL with TiDB
+title: Basic SQL operations
```
Comment (lilin90) [inline basic-sql-operations.md:2]: Please keep the title consistent with the 1st-level heading in the body.
"operations" > "Operations"

File: basic-sql-operations.md [hunk 11]
```diff
@@ -1,174 +1,156 @@
...
 aliases: ['/docs/dev/basic-sql-operations/','/docs/dev/how-to/get-started/explore-sql/']
 ---
 
-# Explore SQL with TiDB
+# Basic SQL Operations
 
 TiDB is compatible with MySQL, you can use MySQL statements directly in most of the cases. For unsupported features, see [Compatibility with MySQL](/mysql-compatibility.md#unsupported-features).
 
-To experiment with SQL and test out TiDB compatibility with MySQL queries, you can [run TiDB directly in your web browser without installing it](https://tour.tidb.io/). You can also first deploy a TiDB cluster and then run SQL statements in it.
+Structured Query Language (SQL) is a declarative language which defines the way that users interact with databases. You can use SQL just as a natural language to talk to the database. This page walks you through the basic SQL statements in TiDB. For a complete list of TiDB statements, see [TiDB SQL Syntax Diagram](https://pingcap.github.io/sqlgram/).
```
Comment (lilin90) [inline basic-sql-operations.md:11]: For this section, I suggest using the existing English version. @ireneontheway @ran-huang If the existing English version is better, just keep them.

File: basic-sql-operations.md [hunk 12]
```diff
@@ -1,174 +1,156 @@
...
+SQL is divided into the following 4 types according to their functions:
 
-### Create a database
+- DDL (Data Definition Language): It is used to define database objects, including databases, tables, views, and indexes.
 
-To create a database, use the `CREATE DATABASE` statement:
+- DML (Data Manipulation Language): It is used to manipulate application related records.
 
-{{< copyable "sql" >}}
+- DQL (Data Query Language)：It is used to query the records after conditional filtering.
```
Comment (lilin90) [inline basic-sql-operations.md:21]: Please do not use Chinese style punctuation in English docs.

File: basic-sql-operations.md [hunk 13]
```diff
@@ -206,71 +190,70 @@ To insert data into a table, use the `INSERT` statement:
...
-+--------+------+------------+
-|      1 | tom  | 2017-10-10 |
-+--------+------+------------+
-```
+> **Note:**
+> 
+> The `UPDATE` and `DELETE` statements without the `WHERE` clause as a filter operate on the entire table.
 
-### Delete data
+DQL is used to retrieve the desired data rows from a table or multiple tables, usually the core content.
```
Comment (lilin90) [inline basic-sql-operations.md:221]: I don't think `usually the core content` is necessary in this sentence. 
> DQL is used to retrieve the desired data rows from a table or multiple tables.

File: basic-sql-operations.md [hunk 14]
```diff
@@ -1,174 +1,156 @@
...
 {{< copyable "sql" >}}
 
 ```sql
 SHOW DATABASES;
 ```
 
-### Delete a database
-
-To delete a database, use the `DROP DATABASE` statement:
+Use the database named `mysql`:
```
Comment (lilin90) [inline basic-sql-operations.md:39]: Please keep the sentence structure consistent with that of others.
Use > To use

**Comment by ireneontheway** [comment]
As I re-clone the repo, this PR is from an unknown repository and cannot be changed. So I close it and make a new PR.: https://github.com/pingcap/docs/pull/3401

**Pull request closed without merging**